tee-worker/omni-executor/client-sdk/packages/client-sdk/src/lib/requests/get-oauth2-google-authorization-url.request.ts

```diff
@@ -0,0 +1,38 @@
+import { assert, compactStripLength, hexToU8a, u8aToString } from '@polkadot/util';
+
+import { JsonRpcRequest } from '@lib/utils/types';
+
+import { enclave } from '@lib/enclave';
+
+/**
+ * Generates an OAuth2 authorization URL for Google authentication.
+ * 
+ * @param args - The arguments object
+ * @param args.googleAccount - The Google account email address
+ * @param args.redirectUri - The URI where Google will redirect after authentication
+ * @returns A Promise that resolves to the Google OAuth2 authorization URL
+ * @throws {Error} If googleAccount or redirectUri is empty or undefined
+ */
+export async function getOAuth2GoogleAuthorizationUrl(args: {
+  googleAccount: string;
+  redirectUri: string;
+}): Promise<string> {
+  const { googleAccount, redirectUri } = args;
+
+  assert((googleAccount?.length ?? 0) > 0, 'Google account is required');
```

Comment:
how about this?
```suggestion
  assert(googleAccount.length > 0, 'Google account is required');
```
The arguments are not optional